tests/integration/llm/prepare.py

```diff
@@ -330,6 +330,13 @@
         "option.max_rolling_batch_size": 4,
         "option.trust_remote_code": True
     },
+    "falcon-11b": {
+        "option.model_id": "tiiuae/falcon-11b",
```

Comment:
nit: it's there in s3 `s3://djl-llm/falcon-11B/`

Reply:
Updated